src/modelmanager.hpp

```diff
@@ -340,6 +340,8 @@ class ModelManager {
      * @brief Updates OVMS configuration with cached configuration file. Will check for newly added model versions
      */
     void updateConfigurationWithoutConfigFile();
+
+    std::string getPipelinesStatusesAsJson();
```

Comment:
ModelManager will now have additional responsibility to generate JSON formatted response. I think this should be part of model_service?